content/en/docs/contribute/_index.md

```diff
@@ -13,68 +13,40 @@ we're happy to have your help! Anyone can contribute, whether you're new to the
 project or you've been around a long time, and whether you self-identify as a
 developer, an end user, or someone who just can't stand seeing typos.
 
-For information on the Kubernetes documentation
- content and style, see the 
- [Documentation style overview](/docs/contribute/style/).
+{{% /capture %}}
 
 {{% capture body %}}
 
-## Types of docs contributors
-
-- A _member_ of the Kubernetes organization who has [signed the CLA](/docs/contribute/start#sign-the-cla)
-  and contributed some time and effort to the project. See
-  [Community membership](https://github.com/kubernetes/community/blob/master/community-membership.md)
-  for specific criteria for membership.
-- A SIG Docs _reviewer_ is a member of the Kubernetes organization who has
-  expressed interest in reviewing documentation pull requests and who has been
-  added to the appropriate GitHub group and `OWNERS` files in the GitHub
-  repository, by a SIG Docs Approver.
-- A SIG Docs _approver_ is a member in good standing who has shown a continued
-  commitment to the project. An approver can merge pull requests
-  and publish content on behalf of the Kubernetes organization.
-  Approvers can also represent SIG Docs in the larger Kubernetes community.
-  Some of the duties of a SIG Docs approver, such as coordinating a release,
-  require a significant time commitment.
+## Getting Started
 
-## Ways to contribute to documentation
+Anyone can open an issue with documentation or contribute a change using a PR,
+but some tasks need a higher level of access in the Kubernetes organization.
```

Comment:
The spatial analogy isn't immediately clear, and Kubernetes community permissions aren't hierarchical enough for a "higher" analogy to automatically make sense.

```suggestion
Some tasks require more trust and need more access in the Kubernetes organization.
```